Kernel/Heap/kmalloc.cpp

```diff
@@ -362,8 +362,11 @@ void* kmalloc(size_t size)
     Thread* current_thread = Thread::current();
     if (!current_thread)
         current_thread = Processor::idle_thread();
-    if (current_thread)
+    if (current_thread) {
+        // FIXME: Is it a bad idea to only panic the kernel after an illegal allocation?
```

Comment:
Not sure why this is a FIXME? Seems appropriate.

Reply:
sure